.circleci/config.yml

```diff
@@ -6,7 +6,53 @@
 #
 version: 2
 jobs:
-  build_test_publish:
+  lint:
+    docker:
+      # Image with python 3.6 and node 8
+      - image: circleci/python:3.6-node-browsers
```

Comment:
This is fine for now, but I think we ought to look into making an image in [mozilla/ci-docker-bases](https://github.com/mozilla/ci-docker-bases/) that provides Python 3.7 and Node 10 or 11. That image could also include tools like Therapist.

Reply:
Filed: https://github.com/mozilla/ci-docker-bases/pull/45